### PR TITLE
Shifting login container

### PR DESCRIPTION
**Pull request by JonlinPei**
1) shift login container and button up on keyboard showing
2) add in secure text entry for password field

### CODE REVIEW

**Comment by JonlinPei** [comment]
I wonder if I did this right.... can you guys see this?  

i also took the liberty of adding the one line for secure text entry on the password field.

File: minigram/IntroViewController.swift [hunk 1]
```diff
@@ -38,14 +45,35 @@ class IntroViewController: UIViewController, UIGestureRecognizerDelegate, UIView
...
+        loginContainer = UIView(frame: CGRect(x: 0, y: 0, width: 320, height: 460))
+        view.addSubview(loginContainer)
+        
+        // SETUP LOGIN CONTAINER SHIFT
+        
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: "keyboardWillShow:", name: UIKeyboardWillShowNotification, object: nil)
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: "keyboardWillHide:", name: UIKeyboardWillHideNotification, object: nil)
+        
+        originalLoginYCenter = loginContainer.center.y
+//        originalSignInButtonsYCenter = signInButtonsContainer.center.y
```
Comment (donokuda) [inline minigram/IntroViewController.swift]: You can remove this if it's not needed

File: minigram/IntroViewController.swift [hunk 2]
```diff
@@ -38,14 +45,35 @@ class IntroViewController: UIViewController, UIGestureRecognizerDelegate, UIView
...
+        
+        var tap:UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: "DismissKeyboard")  //Looks for single or multiple taps.
+//        view.addGestureRecognizer(tap)
```
Comment (donokuda) [inline minigram/IntroViewController.swift]: Same here:

> You can remove this if it's not needed

File: minigram/IntroViewController.swift [hunk 3]
```diff
@@ -38,14 +45,35 @@ class IntroViewController: UIViewController, UIGestureRecognizerDelegate, UIView
...
+        loginContainer.addGestureRecognizer(tap)
+        
+        
+        
         // TITLE
         
+//        introTitle = UILabel(frame: CGRect(x: 116, y: 160, width: 88, height: 28))  // ORIGINAL DIMENSIONS
```
Comment (donokuda) [inline minigram/IntroViewController.swift]: > You can remove this if it's not needed

Comment (edchao) [inline minigram/IntroViewController.swift]: lgtm, if it's redundant feel free.

**Comment by donokuda** [comment]
Looks good. Feel free to remove the commented lines of code before merging.

**Comment by JonlinPei** [comment]
removed those commented lines

**Comment by edchao** [comment]
good job @JonlinPei !

**Pull request merged**